dashboard/app/controllers/aichat_controller.rb

```diff
@@ -41,20 +41,12 @@ def chat_completion
       }
     end
 
-    messages_for_model = params.to_unsafe_h[:storedMessages].filter do |message|
```

Comment:
@bencodeorg - seems like unit tests are passing without `to_unsafe_h` so removed. Lmk what you think. Thanks!

Reply:
huh, [went back](https://github.com/code-dot-org/code-dot-org/commit/97dbf96e7043626ea3c704788fb00e91e52c1594) to where I added this and seems like the tests passed at this point, so maybe I totally misunderstood something at the time. Also did a quick verify that if I changed the expect condition to something else, it fails, so it seems like it's doing its job. Thanks for cleaning up!